### DISCUSSION TITLE
Channel/tab Filter Box

### DISCUSSION THREAD

**Discussion by Coldblackice** [Chatterino/chatterino2#3598] [Ideas]
A helpful ability would be a quick-filter textbox for tabs, a static textbox at the bottom of the tab bar that functioning as a "Quick filter". As the user types into the box, Chatterino hides tabs that aren't contained in the filter, showing only those that match.

Or less developmentally intensive, Chatterino instead highlighting channel tabs that match the filter input.

Input focus into the box could be initiated by keyboard shortcut.

**Comment by jupjohn** [comment]
I'm unsure on what you're trying to achieve. Is it to filter tabs down visually or are you searching for specific tabs?

If you're just wanting to search then try `Ctrl + K`
